components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/constant/ConfigurationConstants.java

```diff
@@ -121,9 +121,7 @@
         ERROR_CODE_FILE_DOES_NOT_EXISTS("CONFIGM_00043", "File with the id: %s does not exists."),
         ERROR_CODE_FILES_DOES_NOT_EXISTS("CONFIGM_00044", "Resource: %s does not contain any files."),
         ERROR_CODE_FILE_IDENTIFIERS_REQUIRED("CONFIGM_00045", "File identifier validation failed for " +
-                "one or more file identifiers in: %s"),
-        ERROR_CODE_GET_FILES_BY_TENANT_ID("CONFIGM_00046", "Resource type: %s does not contain any files for the "
-                + "tenant id: %s");
+                "one or more file identifiers in: %s");
```

Comment:
Fullstop is missing from the error message

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2690